.github/workflows/reusable_basic.yml

```diff
@@ -124,7 +135,19 @@ jobs:
     - name: Install apt packages
       run: |
         sudo apt-get update
-        sudo apt-get install -y clang cmake libnuma-dev lcov
+        sudo apt-get install -y clang libnuma-dev lcov
+
+    - name: Install cmake (latest)
+      if: matrix.cmake_ver == 'latest'
+      run: |
+        sudo apt-get install -y cmake
+
+    - name: Install cmake (minimum supported version)
+      if: matrix.cmake_ver != 'latest'
+      run: |
+        wget https://github.com/Kitware/CMake/releases/download/v${{matrix.cmake_ver}}/cmake-${{matrix.cmake_ver}}-Linux-x86_64.sh
+        chmod +x cmake-${{matrix.cmake_ver}}-Linux-x86_64.sh
+        sudo ./cmake-${{matrix.cmake_ver}}-Linux-x86_64.sh --skip-license --prefix=/usr/local
```

Comment:
If you want to install CMake, first of all, remove the previously image-installed CMake.

Reply:
done